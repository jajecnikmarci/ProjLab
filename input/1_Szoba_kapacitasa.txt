starttest
add room room1 3
add room room2 3
add person room1 Student student1
add person room1 Professor professor1
add person room1 Cleaner cleaner1
add person room2 Student student2
add door room1 room2 door1
printstate room1
printstate room2
gotoroom student2 room1
printstate room1
printstate room2
endtest