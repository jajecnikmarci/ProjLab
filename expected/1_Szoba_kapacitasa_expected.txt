room1
	- capacity: 3
	- people: 
		student1
		professor1
		cleaner1
	- doors: 
		door1
	- items: 
	- poisonEffects: 
	- stunEffects: 
	- lastCleaning: null
room2
	- capacity: 3
	- people: 
		student2
	- doors: 
		door1
	- items: 
	- poisonEffects: 
	- stunEffects: 
	- lastCleaning: null
room1
	- capacity: 3
	- people: 
		student1
		professor1
		cleaner1
	- doors: 
		door1
	- items: 
	- poisonEffects: 
	- stunEffects: 
	- lastCleaning: null
room2
	- capacity: 3
	- people: 
		student2
	- doors: 
		door1
	- items: 
	- poisonEffects: 
	- stunEffects: 
	- lastCleaning: null
